### PR TITLE
Pull Request for issues #512

### PR DESCRIPTION
**Pull request by ojrb**
Hi @e0i this PR solve the issue #512, I explain in the **MountPath** section the use of the Juntion function, to solve the Directory Folder problem

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/startlayout/import-startlayout.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+When you use this switch, to indicate for example the mount path to the folder **C:\Images**, the command inidcates that a folder **C:\ImagesUsers** does not exist and  generates an error when executes, the correct expected folder is **C:\Images\Users**.  To solve this problem you can use the [Junction](https://docs.microsoft.com/sysinternals/downloads/junction) function, to create the expected **C:\ImagesUsers** junction targeted to the folder **C:\Images\Users**
```
Comment (nenonix) [inline docset/windows/startlayout/import-startlayout.md]:
```suggestion
When you use this switch, for example to indicate the mount path to the folder **C:\Images**, the command indicates that a folder **C:\ImagesUsers** does not exist and generates an error when executed. The correct expected folder is **C:\Images\Users**.  To resolve this, download [Junction](https://docs.microsoft.com/sysinternals/downloads/junction) to create the expected **C:\ImagesUsers** junction targeted to the folder **C:\Images\Users**.
```

**Comment by ojrb** [comment]
Hi @e0i I accepted the suggestion of @nenonix  and committed to the PR, Please review and Merge !

**Comment by e0i** [comment]
@officedocsbot close

**Pull request closed without merging**